avocado/core/nrunner.py

```diff
@@ -542,11 +542,10 @@ class PythonUnittestRunner(BaseRunner):
 
     Runnable attributes usage:
 
-     * uri: a "dotted name" that can be given to Python standard
-            library's :meth:`unittest.TestLoader.loadTestsFromName`
-            method. While it's not enforced, it's highly recommended
-            that this is "a test method within a test case class" within
-            a test module.  Example is: "module.Class.test_method".
+     * uri: a test reference (string that can be resolved into one or more
+            tests).  It's highly recommended that this is "a test method within
```

Comment:
A runner is responsible for running one test, so the affirmative "string that can be resolved into one or more tests" is not true. The runner should receive one single test reference on its uri parameter.